telegram/user.py

```diff
@@ -44,7 +44,14 @@ class User(TelegramObject):
         bot (Optional[Bot]): The Bot to use for instance methods
     """
 
-    def __init__(self, id, first_name, type='', last_name='', username='', bot=None, **kwargs):
+    def __init__(self,
+                 id,
+                 first_name,
+                 type=None,
```

Comment:
Hmm.. I don't think it's ever been there tbh.. I might be us who made an error and confused User for Chat or something.